src/database/seeders/20191028063944-demo-categories.js

```diff
@@ -1,4 +1,4 @@
-module.exports = {
+export default {
```

Comment:
This parsing error seems to be an eslint error. This can be corrected  by adding this to the parseOptions field of the .eslintrc.js file: 

`"ecmaVersion": 11`

or 

`""ecmaVersion": 6,"`

Should work